src/installer/installer.go

```diff
@@ -483,14 +483,18 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 	}
 	i.UpdateHostInstallProgress(models.HostStageWaitingForControlPlane, waitingForMastersStatusInfo)
 
+	hasValidvSphereCredentials := common.HasValidvSphereCredentials(ctx, i.inventoryClient, i.log)
+	if hasValidvSphereCredentials {
+		i.log.Infof("hasValidvSphereCredentials: %v", hasValidvSphereCredentials)
```

Comment:
nit: with an info-level log should the msg be more user friendly? There are a few of these scattered around and thought they were perhaps leftover from debugging